fflas-ffpack/fflas/fflas_faxpy.inl

```diff
@@ -172,14 +172,38 @@ namespace FFLAS { namespace details {
             }
             else
             {
-                typename Field::Element_ptr Xc = fflas_new (F,N) ;
-                typename Field::Element_ptr Yc = fflas_new (F,N) ;
-                fassign(F,N,X,incX,Xc,1);
-                fassign(F,N,Y,incY,Yc,1);
-                faxpy(F,N,a,Xc,1,Yc,1,FieldCategories::ModularTag());
-                fassign(F,N,Yc,1,Y,incY);
-                fflas_delete(Xc);
-                fflas_delete(Yc);
+                typename Field::Element_ptr Xc;
```

Comment:
Define Xc as ConstElement_ptr instead, which will remove the need for const_cast

Reply:
Cf. our offline discussion, the const_cast is needed for compatibility with the signature of `faxpy`. The proposed fix doesn't work (as is) since there is a branch where `Xc` is assigned